### PR TITLE
node: arbitrum watcher fix

### PR DESCRIPTION
**Pull request by bruce-riley**
Somehow this change to poller.go did not get committed with the previous PR.

### CODE REVIEW

**Review by evan-gray — DISMISSED** [review]
Seems reasonable. Should there be a comment that L1BlockNumber is part of the Arbitrum spec?

File: node/pkg/watchers/evm/connectors/poller.go [hunk 1]
```diff
@@ -171,9 +171,9 @@ func (b *BlockPollConnector) getBlock(ctx context.Context, logger *zap.Logger, n
 	}
 
 	type Marshaller struct {
-		Number     *ethHexUtils.Big
-		Hash       ethCommon.Hash `json:"hash"`
-		Difficulty *ethHexUtils.Big
+		Number        *ethHexUtils.Big
+		Hash          ethCommon.Hash `json:"hash"`
+		L1BlockNumber *ethHexUtils.Big
```
Comment (hendrikhofstadt) [inline node/pkg/watchers/evm/connectors/poller.go:176]: Let's add a comment saying this field is only populated in Arbitrum

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]

File: node/pkg/watchers/evm/connectors/poller.go (hunk 1 above)
Comment (bruce-riley) [reply node/pkg/watchers/evm/connectors/poller.go]: Comment added.

**Review by evan-gray — DISMISSED** [review]

File: node/pkg/watchers/evm/connectors/poller.go (hunk 1 above)
Comment (hendrikhofstadt) [reply node/pkg/watchers/evm/connectors/poller.go]: Please make it a godoc string such that it shows up in IDE autocompletion.

E.g.
```
// L1BlockNumber is the L1 block number an Arbitrum batch containing this block was submitted. This field is only populated when connecting to Arbitrum
L1BlockNumber *ethHexUtils.Big
```

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]
One more nit on the comment

**Review by evan-gray — APPROVED** [review]

**Review by hendrikhofstadt — APPROVED** [review]

File: node/pkg/watchers/evm/connectors/poller.go (hunk 1 above)
Comment (bruce-riley) [reply node/pkg/watchers/evm/connectors/poller.go:179]: Done.

**Pull request merged**